src/lib/molecules/agenda-card.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+	import { Button } from '$lib/index.js';
+
+	export let cardEvent, cardDate, cardAddress, cardPrice, cardImage;
+</script>
+
+<section style="background-image: url({cardImage});">
+	<div class="data">
```

Comment:
Als je toch een heading toevoegt, bijvoorbeeld in div.data, dan kan je er wellicht ook een header van deze div maken. De classname _data_ is sowieso te algemeen, maar dan ook niet meer nodig?!